src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.MultiAgent.cs

```diff
@@ -565,6 +571,13 @@ private static size_t CurlReceiveHeadersCallback(IntPtr buffer, size_t size, siz
                                 {
                                     response.Content.Headers.TryAddWithoutValidation(headerName, headerValue);
                                 }
+                                else
+                                {
+                                    if (easy._isRedirect && string.Equals(headerName, HttpKnownHeaderNames.Location))
```

Comment:
And is this the right equality comparison?  I'm wondering if it should be using OrdinalIgnoreCase?